weld/sir.rs

```diff
@@ -441,6 +461,9 @@ fn sir_param_correction_helper(prog: &mut SirProgram,
                 Negate { ref child, .. } => {
                     vars.push(child.clone());
                 }
+                Broadcast { ref child, .. } => {
```

Comment:
I put a note on what this does in `ast.rs`; putting in `sir.rs` seems kind of weird.